py/openage/codegen/codegen.py

```diff
@@ -6,7 +6,7 @@
 
 from openage.log import dbg
 
-contenttemplate = """\
+contenttemplate = """
```

Comment:
Fixed & rebased.

The alternative would be to add `.strip()` at the end. But I find the backslash cleaner.